src/navigators/BottomNavigationBar.tsx

```diff
@@ -56,7 +56,7 @@ export const BottomNavigationBar = ({
             ? options.tabBarLabel
             : options.title !== undefined
             ? options.title
-            : route.title;
+            : route.name;
```

Comment:
After the upgrade the title property was removed. We now default to name, however, in practice this should never happen.